Parsers/Pole Command with Vote Tracking.js

```diff
@@ -0,0 +1,55 @@
+/*
+activation_example: !poll [question] with [option1], [option2], ...
+regex: !poll (.+) with (.+)
+flags: gmi
+order: 102
+stop_processing: true
+active: true
+*/
+
+// In-memory storage for polls (you could use a database for persistence)
+var polls = {};
```

Comment:
This is for a production application. We are not accepting parsers that are not persistent or come with the app updates that would enable persistence. 

Additionally the slack already has a polling feature